src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -28,6 +30,22 @@
 
 FILTER_HAS_A_REPLAY = "AND !replayId:''"
 
+RESPONSE_EXAMPLES = [
```

Comment:
let's move this example into `examples/replay_examples.py` ex https://github.com/getsentry/sentry/blob/5822d6b40bc7952360f3083ef9d33f8ede2f41a1/src/sentry/apidocs/examples/replay_examples.py#L44